### PR TITLE
Editation of first post, tighter syntax, links to stuff in second hal…

### PR DESCRIPTION
**Pull request by pnadeausfdc**
…f, what the hell was I on to let that first draft escape?

### CODE REVIEW

**Comment by pnadeausfdc** [comment]
@dextermilo Can you have a look? Post 2 coming soon.

File: _posts/2017-11-07-post-1-a-brief-history-of-ai.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+layout:   post
+title:    "Post 1: A Brief History of AI"
```
Comment (dextermilo) [inline _posts/2017-11-07-post-1-a-brief-history-of-ai.md]: @pnadeausfdc does it matter if the title here, and filename, don't match the first heading?

Comment (dextermilo) [reply _posts/2017-11-07-post-1-a-brief-history-of-ai.md]: @pnadeausfdc my biggest concern with the filename changing is that the URL to the post will also change

**Comment by pnadeausfdc** [comment]
Alright, I will put it back

**Pull request merged**